deps/tls/common.lua

```diff
@@ -393,9 +393,9 @@ end
 exports.TLSSocket = TLSSocket
 
 -------------------------------------------------------------------------------
-local VERIFY_PEER = { "peer" }
-local VERIFY_PEER_FAIL = { "peer", "fail_if_no_peer_cert" }
-local VERIFY_NONE = { "none" }
+local VERIFY_PEER = openssl.ssl.peer
+local VERIFY_PEER_FAIL = openssl.ssl.fail
```

Comment:
hi, ssl.peer can be bit combine with ssl.fail or ssl.once, VERIFY_PEER_FAIL should be ssl.fail+ssl.peer(bit.bor(ssl.fail,ssl.peer)